ansible/playbook-setup-quince.yml

```diff
@@ -26,7 +26,11 @@
     package:
       state: present
       name:
+    # Maintenance
+      - dnf-automatic
+      - python3-tracer
     # Debugging tools
+      - bash-completion
```

Comment:
Why is `bash-completion` needed for this?

Reply:
I noticed absence of `bash-completion` while paying with systemd. I can drop this or make a dedicated commit — what do you prefer?

Reply:
Yes, please put it in a separate commit if it's unrelated.

Another question, in #7 I put the common file in a new directory `files/common/` while your files are directly in `files/`. Presumably we should unify this? (I don't have any strong preference for either.)

Reply:
@ulm good point. With the current theme, it takes (minimal but) effort to classify "common" as a "magic" host, as a non-host. After consideration, I'd prefer without "common".

Reply:
> Yes, please put it in a separate commit if it's unrelated.

@ulm done, pushed